app/models/user.rb

```diff
@@ -35,6 +35,7 @@ class User < ActiveRecord::Base
          :password_archivable, # in signonotron2/lib/devise/models/password_archivable.rb
          :password_expirable   # in signonotron2/lib/devise/models/password_expirable.rb
 
+  validates :require_2sv, inclusion: [false], if: ->(u) { u.otp_secret_key.present? }
```

Comment:
:see_no_evil: I missed the resetting of the flag. Good spot.

Addressed in 09b6bdd and 605023d